### ISSUE TITLE
iframe embed should include title attribute

### ISSUE THREAD

**Issue by jronallo** [UniversalViewer/universalviewer#637]
For accessibility the iframe presented in the share embed should include a title attribute with the title from the manifest if present.

**Comment by LlGC-szw** [comment]
All issues will be triaged for further investigation or closure by the 28 September 2023. If your issue is still relevant and would like for it be investigated further please comment by 14 September 2023.

**Comment by demiankatz** [comment]
As of this writing, embed iframe still lacks a title attribute.
